tools/editor/editor_settings.cpp

```diff
@@ -575,6 +575,8 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	set("scenetree_editor/start_create_dialog_fully_expanded",false);
 	set("scenetree_editor/draw_relationship_lines",false);
 	set("scenetree_editor/relationship_line_color",Color::html("464646"));
+	set("scenetree_editor/new_node_name_casing",0);
```

Comment:
I agree that it should probably be a project setting instead of an editor setting actually. @bojidar-bg  Any pointer on where it should be declared then?